src/pages/trails/ballots/view/ballot-view.vue

```diff
@@ -258,36 +273,30 @@ export default {
                   @clickBtn="cancel()"
                 )
         q-card-section().q-pb-none.cursor-pointer.statics-section.statics-section-620
-          div.text-section.column(@click="showVoters()")
-            div.statics-section-item(v-if="ballot.total_voters > 0")
-              span.text-weight-bold {{ getPercentofTotal(getWinner) }}%&nbsp
-              span.opacity06  {{ getWinner.key.toUpperCase() }} {{ getLoser.key ? ` lead over ${getLoser.key.toUpperCase()}` : ` lead over others` }}
+          div.text-section.column
+            div(v-if="ballot.total_voters > 0")
+              span.statisctic-title Most voted
+              div.statisctic-body
+               span.text-weight-bold  {{ getWinner.key.toUpperCase() }}
+               span.text-weight-bold {{ getPercentofTotal(getWinner) }}%&nbsp
+            div(v-if="ballot.total_voters > 0")
+              span.statisctic-title Number of accounts
+              div.statisctic-body(@click="showVoters()")
+               span.text-weight-bold  {{ ballot.total_voters }}
+               div.poll-icon.details
             div.statics-section-item(v-else)
               span  {{ getWinner }}
-            div.statics-section-item
-              span.text-weight-bold {{ `${ballot.total_voters} Accounts` }}
-              span.opacity06 &nbspvoted
-            div.statics-section-item
-              span.text-weight-bold {{ ballot.total_raw_weight.split(' ')[0].split('.')[0] }}&nbsp
-              span.opacity06 {{ ballot.total_raw_weight.split(' ')[1]  }}&nbsp
-              span.opacity06 tokens
-            div.statics-section-item(v-if="ballot.proposal_info")
-              span.text-weight-bold {{ getRequestAmountRounded(ballot.proposal_info.total_requested) }}&nbsp
-              span.opacity06 {{ $t('pages.trails.ballots.requestAmount') }}
-
+            div
+              span.statisctic-title Number of tokens
+              div.statisctic-body
+                span.text-weight-bold {{ ballot.total_raw_weight.split(' ')[0].split('.')[0] }}&nbsp
+                span.text-weight-bold {{ ballot.total_raw_weight.split(' ')[1]  }}&nbsp
```

Comment:
maybe create a method/property to parse the desired weight and return as a single value in a single span?